yt_dlp/extractor/zaiko.py

```diff
@@ -0,0 +1,71 @@
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+    extract_attributes,
+    remove_start,
+    unescapeHTML,
+    url_or_none,
+    str_or_none,
+    int_or_none,
+)
+
+
+class ZaikoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?zaiko\.io/event/(?P<id>\d+)/stream(?:/\d+)+'
+    _TESTS = [{
+        'url': 'https://zaiko.io/event/324868/stream/20571/20571',
+        'info_dict': {
+            'id': '324868',
+            'ext': 'mp4',
+            'title': 'ZAIKO STREAMING TEST',
+
+            'alt_title': '[VOD] ZAIKO STREAMING TEST_20210603(Do Not Delete)',
+            'uploader_id': '454',
+            'uploader': 'ZAIKO ZERO',
+            'timestamp': 1583809200,
+            'thumbnail': r're:https://[a-z0-9]+.cloudfront.net/[a-z0-9_]+/[a-z0-9_]+',
+            'upload_date': '20200310',
+            'categories': ['Tech House'],
+        }
+    }]
+
+    def _parse_vue_element_attr(self, name, string, video_id):
+        page_elem = self._search_regex(rf'(<{name}[^>]+>)', string, name)
+        attrs = {}
+        for key, value in extract_attributes(page_elem).items():
+            if key[0] == ':':
+                attrs[remove_start(key, ':')] = self._parse_json(
+                    value, video_id, transform_source=unescapeHTML, fatal=False)
+            else:
+                attrs[key] = value
```

Comment:
Ah I see now, not all of the attributes on the player page are JSON. My bad.
It doesn't look like any of the non-JSON attrs will ever be useful, so I think we can do this:
```suggestion
            if key.startswith(':'):
                attrs[key[1:]] = self._parse_json(
                    value, video_id, transform_source=unescapeHTML, fatal=False)
```

Reply:
agreed